docs/CHANGELOG.md

```diff
@@ -12,6 +12,16 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Added
 
 - New `EVM.NodePool.SelectionMode` `TotalDifficulty` to use the node with the greatest total difficulty.
+- `EVM.NodePool.SyncThreshold` to ensure that live nodes do not lag too far behind.
+
+> ```toml
+> SyncThreshold = 5 # Default
```

Comment:
We may want different chain-specific defaults. I think on mainnet 2-3 is more reasonable. Matic probably need to be bumped to say 10 or so.

Reply:
Yeah, do you think we should default to 0 for disabled until we can calibrate each chain?

Reply:
I dont think that's necessary, I'd say just make your best guess based on block time.

Reply:
5 probably ok for a default then override in select cases

Reply:
Updated `SyncThreshold = 10` for 'fast' chains: poly, arb, opt, metis, bsc